jakttest/README.md

```diff
@@ -3,18 +3,7 @@
 Jakttest is the test driver we use to test the Jakt compiler with code snippets.
 It's also written in Jakt.
 
-## Build dependencies
-
-To build Jakttest you should use the `ninja` build system which will use the provided `build.ninja` file.
-It needs the same `clang++` that is required to build any other Jakt file, and `python` 3.6+ to run the
-patch script that patches the generated C++ from Jakt code to accept the external C++ code.
-
-As a list:
-
-- `ninja` (or `ninja-build` on some package managers)
-- `clang++`
```

Comment:
This should be changed to saying we require `cmake` and `clang++` rather than being removed, with the text in the paragraph changed to match.

Reply:
That seems redundant, because Jakttest is built with Jakt anyway.

Reply:
just following up from discord chat - the README here should change to point people to the main README for build instructions

Reply:
How do the current changes look?